src/Perspex.Controls/Primitives/Popup.cs

```diff
@@ -283,6 +283,17 @@ private Point GetPosition()
                     return target?.PointToScreen(new Point(0, target.Bounds.Height)) ?? zero;
                 case PlacementMode.Right:
                     return target?.PointToScreen(new Point(target.Bounds.Width, 0)) ?? zero;
+                case PlacementMode.Center:
+                    if ((target == null) || (Child == null))
+                    {
+                        return zero;
+                    }
+                    else
+                    {
+                        Child.Measure(target.Bounds.Size);
```

Comment:
This still doesn't look correct - the size of `target` and `Child` are unrelated so I don't understand why one is being measured with the other's size.